### DISCUSSION TITLE
JK_B2A20S20P HV: 11.XW SV: 11.31H help me

### DISCUSSION THREAD

**Discussion by przybora** [syssi/esphome-jk-bms#483] [Q&A]
Friends, I have no strength left. I tried with Wemos d1 mini and ESP32 as in the attached photo. I can't download the data, I've already tried different versions from the ESP32 list... nothing works :( Can someone help me?
I have JK_B2A20S20P HV: 11.XW SV: 11.31H
Could someone help me prepare a file for WSP-WROOM-32? Or under Womos D1 mini ESP2688?

![20240319_165627](https://github.com/syssi/esphome-jk-bms/assets/22376161/fdb7d618-3e2c-457f-823b-2e740f2b3bd6)
![Screenshot_20240316_155918_BMS](https://github.com/syssi/esphome-jk-bms/assets/22376161/a87204e5-4869-494a-8c99-3c87d95df415)
[logs_j-k_run.txt](https://github.com/syssi/esphome-jk-bms/files/14654163/logs_j-k_run.txt)

**Comment by przybora** [comment]
INFO ESPHome 2024.2.2
INFO Reading configuration /config/esphome/bms.yaml...
INFO Updating https://github.com/syssi/esphome-jk-bms.git@main
Failed config

esp32: [source /config/esphome/bms.yaml:52]
  
  This board is unknown, please set the variant manually.
  board: wesp32doit-devkit-v1
  framework: 
    type: arduino

**Reply by syssi** [reply]
Please try to use the YAML example without changing the board setting. This should work out of the box if you change the `mac_address` to the one of your BMS or do you want to use a wired connection?
